CONTRIBUTING.md

```diff
@@ -0,0 +1,8 @@
+# Instructions for new contributors 
+
+ - Make a fork of https://github.com/erikras/redux-form
+ - ? Install deps ?
+ - Change the code to implement your feature/fix
+ - ? Rebuild the app ?
```

Comment:
pre-commit hooks should do a sanity-check build; other good steps to take before making a PR are to run a typechecker (`npm run test:flow` - quick but not thorough) and then the full test suite (`npm run test`). Another good thing to check is `npm run test:cov` - this will check if you've added any new code that is not exercised by the test suite.

Reply:
I've improved the docs. Thanks for the feed back @asazernik !